src/features/Apiexplorer/Schema/RecursiveContent/StreamTypesObject/StreamTypesBody.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styles from '../../Schema.module.scss';
+
+type TStreamTypesBody = {
+  type: string;
+  _enum: Array<
+    | 'balance'
+    | 'candles'
+    | 'cashier_payments'
+    | 'p2p_advert'
+    | 'p2p_advertiser'
+    | 'p2p_order'
+    | 'proposal'
+    | 'proposal_open_contract'
+    | 'ticks'
+    | 'transaction'
+    | 'trading_platform_asset_listing'
+    | 'website_status'
+    | 'p2p_settings'
+    | 'crypto_estimations'
+  >;
+};
```

Comment:
Can we grab these from the API types itself so that if the backend changes we don't have change it again